src/main/resources/db/migration/V1__create_breweries_table.sql

```diff
@@ -0,0 +1,18 @@
+CREATE EXTENSION IF NOT EXISTS postgis;
+CREATE EXTENSION IF NOT EXISTS pg_trgm;
+
+CREATE SCHEMA openbrewerydb
+
+CREATE TABLE IF NOT EXISTS openbrewerydb.breweries (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    brewery_type VARCHAR,
+    city VARCHAR NOT NULL,
+    state VARCHAR NOT NULL,
+    country VARCHAR NOT NULL,
+    location geography(POINT,4326),
```

Comment:
This is a postgis geometry.

Reply:
What's the 4326?

Reply:
It's the code to use WGS84: https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84

Reply:
This helped too: https://gis.stackexchange.com/questions/367366/is-the-geometry4326-an-alias-for-geography-datatype

Reply:
👍